playbooks/code-review.md

```diff
@@ -2,3 +2,45 @@
 title: Code reviews
 description: How our code review process works 
 ---
+
+Reviewing other people's code is a responsibility of every single developer. Instead of viewing it as a hassle, it should be looked upon as a necessary process in order to keep the code clean and consistent.
+It is also an opportunity to learn something new.
+
+Rules on..
+
+## How a Code Review process should look like
+
+..are written below.
+
+### Terms:
+ - PR - Pull Request.
+ - PR Creator - a person who initiated the PR.
+ - Reviewer - a person assigned to do a PR code review.
+
+### Rules:
+1. Every PR has to be reviewed, with the exception of fix/hotfix PR-s that can be merged by the code owner on his own responsibility.
+2. When creating a PR, you should select **EXACTLY ONE** person as a reviewer. This person has to be someone who is working at the moment (not on vacation). 
+If there is someone else who should see this PR, he/she should be assigned as Assignee on this PR.
```

Comment:
It would be good to mention the difference between assignee an reviewer, since these are 2 different roles on a PR

Reply:
In this flow, an Assignee is just an "interested party", someone who can review, but doesn't have to. Assignee has zero responsibility for a PR

Reply:
Agreed with @ibrcko. 
Assignee is a keyword in our code review lingo and should be well defined to remove any ambiguity about the roles and responsibilities of an Assignee

Reply:
```suggestion
If there is someone else who should see this PR, he/she should be assigned as Assignee on this PR. An Assignee is just an "interested party", someone who can review, but doesn't have to and has zero responsibility for a PR.
```